### ISSUE TITLE
Feature Request: Rearrange

### ISSUE THREAD

**Issue by OssNass** [nsz32/docklike-plugin#33]
Is it possible to add the ability to rearrange pinned apps by drag and drop ?

**Comment by OssNass** [comment]
Never mind, found it

**Comment by nsz32** [comment]
No problem,
In fact, the plugin needs a basic help section (and to be shown at first launch).
So your question is quite significant !
For now, I reopen it with label good first issue
TY,

> To rearrange buttons, use CTLR+mouse drag

**Comment by coreybruce** [comment]
Yeah really does need the ability to be able to rearrange the order of icons, quite tedious and in a way clunky to have to open everything I want to pin in order, would really appreciate this feature!

**Comment by okamsn** [comment]
> Yeah really does need the ability to be able to rearrange the order of icons, quite tedious and in a way clunky to have to open everything I want to pin in order especially when I want to add things later and would have to close things to arrange it, would really appreciate this feature! :D

This is already possible. You can re-arrange the icons by holding the Control key and dragging the icons with the mouse.

**Comment by coreybruce** [comment]
> > Yeah really does need the ability to be able to rearrange the order of icons, quite tedious and in a way clunky to have to open everything I want to pin in order especially when I want to add things later and would have to close things to arrange it, would really appreciate this feature! :D
> 
> This is already possible. You can re-arrange the icons by holding the Control key and dragging the icons with the mouse.

Ahh I see, should really just be movable by simply dragging as they would make it easier for the user out of the box as that's how all of them work with the users experience on Linux, Windows and Mac OS. You could than add a lock icon option or even a option to switch between drag rearrange and ctrl click rearrange.

I think those would be really good additions for the user. 🙂

**Comment by coreybruce** [comment]
Actually I will make a new separate post about this request.
